labs/lab-4-jquery/lab-4.js

```diff
@@ -0,0 +1,54 @@
+class Party {
+  constructor(name, contact, size) {
+    this.name = name;
+    this.contact = contact;
+    this.size = size;
+  }
+}
+
+class Table {
+  constructor(available, number){ // removed position as constructor method
+    this.available = true;
+    this.number = number;
+    // this.position = position;
+    this.party = null;
+  }
+  reserveTable(name, contact, size) {
+    this.party = new Party(name, contact, size);
+    this.available = false;
+  }
+  render() {
+    $(".room-container").append(`<button class="table">${this.number}</button>`);
```

Comment:
You can create an element separately from appending it, like this:

```javascript
var button = $(`<button class="table">${this.number}</button>`);
$(".room-container").append(button);
```

That way you can also add event hooks to your button when you render it:

```javascript
button.click(function() {
  console.log("I have been clicked on! I am table", this.number);
});
```